extensions/wikia/Flags/scripts/FlagsModal.js

```diff
@@ -94,9 +94,15 @@ require(
 				var template;
 
 				loader.processStyle(res.styles);
-				template = res.mustache[0];
 
-				flagsData[0].flags = prepareFlagsData(flagsData[0].flags);
+				if (flagsData[0].flags) {
+					template = res.mustache[0];
+					flagsData[0].flags = prepareFlagsData(flagsData[0].flags);
+				} else if (flagsData[0].emptyMessage) {
+					template = res.mustache[1];
+				} else if (flagsData[0].exceptionMessage) {
+					template = res.mustache[2];
+				}
```

Comment:
OK, remember about that one for the future as code in anonymous functions is harder to debug